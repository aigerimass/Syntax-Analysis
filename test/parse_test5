Func_with_many_args(a; b; c; d; e;){
    return a + b + c + d + e;
}
F(){
}
Main () {
   v = Func_with_many_args(1+12; 1; 1; 1;1;1;);
}
