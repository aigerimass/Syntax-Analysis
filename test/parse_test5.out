----------SYNTAX-ANALYSIS----------
Function Definition: [
name = "Func_with_many_args",
args: "a", "b", "c", "d", e
body:
>Return(Plus(Plus(Plus(Plus(Var "a", Var "b"), Var "c"), Var "d"), Var "e"));

]
--------
Function Definition: [
name = "F",
args: 
body:
]
--------
Function Definition: [
name = "Main",
args: 
body:
>Bind(Var "v", >Call(name: "Func_with_many_args", args: Plus(1, 12), 1, 1, 1, 1, 1));

]
--------
----------INTERVAL-ANALYSIS----------
Func_with_many_args (a; b; c; d; e) {
#a = [13, 13]
#b = [1, 1]
#c = [1, 1]
#d = [1, 1]
#e = [1, 1]
return a + b + c + d + e;
}
------------
F () {
}
------------
Main () {
#v = [17, 17]
v = Func_with_many_args (1 + 12;1;1;1;1;1);
#v = [17, 17];
}
------------

